packages/gatsby-source-contentful/src/gatsby-node.js

```diff
@@ -2,11 +2,13 @@ const path = require(`path`)
 const isOnline = require(`is-online`)
 const _ = require(`lodash`)
 const fs = require(`fs-extra`)
+const { createClient } = require(`contentful`)
+
 const normalize = require(`./normalize`)
 const fetchData = require(`./fetch`)
 const { createPluginConfig, validateOptions } = require(`./plugin-options`)
 const { downloadContentfulAssets } = require(`./download-contentful-assets`)
-const { createClient } = require(`contentful`)
+
```

Comment:
Keep it but I'm curious whether there's a particular reason for moving the import up. It implies it's relevant for side effects and I'm hoping it's not :)

Reply:
We need it for resolving, so we need it always. Dependencies I need in all cases I do require on top. Would you require "on runtime" down there in line 200 when the client is created? 🤔